ts_commands_i_test.go

```diff
@@ -429,7 +430,7 @@ func TestTsDeleteRow(t *testing.T) {
 		t.Fatal(err.Error())
 	}
 	if err = cluster.Execute(cmd); err != nil {
-		// TODO: remove Skip when TS is merged
+		// TODO FUTURE: remove Skip when TS is merged into KV
```

Comment:
That's how it used to work. I want it to always run the tests if the bucket type is present, otherwise skip.